docs/core/tools/cli-msbuild-architecture.md

```diff
@@ -13,7 +13,7 @@ ms.assetid: 7fff0f61-ac23-42f0-9661-72a7240a4456
 
 # High-level overview of changes in the .NET Core tools
 
-This document will describe in high-level the changes that moving from *project.json* to MSBuild and *.csproj* project system bring. It will outline the new way the tooling is layered all-up and which new pieces are available and what is their place in the overall picture. After reading this article, you should have a better understanding of all of the pieces that make up .NET Core tooling after moving to MSBuild and *.csproj*. 
```

Comment:
I do like ".NET Core SDK 1.0." Let's see which way they want to go, and we'll polish it up as needed. As a frame of reference tho, the ".NET Core 1.0.4/1.1.1" mention can sit behind the phrase "that occurred with the release of," since I'm not implying anything special other than "these all came out at the same time." We'll see how they want to play it.